### PR TITLE
changed LoadTestTimerConfig and MeshAdapterConfigComponent to functional component

### LINKED ISSUE #8780: [UI] convert `LoadTestTimeConfig` and `MeshAdapterConfigComponent` from class based to functional component
### Current Behavior
`LoadTestTimeConfig` and `MeshAdapterConfigComponent` a class-based components. 

### Desired Behavior
`LoadTestTimeConfig` and `MeshAdapterConfigComponent` should be functional components.

### Implementation
Convert all the Class-based components to new updated functional components. It is important here to convert a class-based component to a functional component you may need to convert some functions as well. Check if the function is used in other components or not.

Also, make sure to use the hooks while converting them. For example, you need to covert withNotify with useNotification, useRouter hook. Please check if a hook for functional components is available and use that.

Here is a quick blog that will help you get started with : [Link](https://www.digitalocean.com/community/tutorials/five-ways-to-convert-react-class-components-to-functional-components-with-react-hooks)

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by EraKin575**
Signed-off-by: EraKin575 <tejaskumar574@gmail.com>

**Notes for Reviewers**

This PR fixes #8780

Changed to functional component

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by sudhanshutech** [comment]
@EraKin575 are you also taking care of `MeshAdapterConfigComponent` in this PR

**Comment by EraKin575** [comment]
@sudhanshutech  yes

File: ui/components/MeshAdapterConfigComponent.js [hunk 1]
```diff
@@ -11,17 +11,68 @@ import ReactSelectWrapper from "./ReactSelectWrapper";
 import { updateAdaptersInfo, updateProgress } from "../lib/store";
 import dataFetch from "../lib/data-fetch";
 import changeAdapterState from './graphql/mutations/AdapterStatusMutation';
-import { withNotify } from "../utils/hooks/useNotification";
+import { withNotify,useNotification } from "../utils/hooks/useNotification";
 import { EVENT_TYPES } from "../lib/event-types";
 import BadgeAvatars from './CustomAvatar';
 
-const styles = (theme) => ({
+// const styles = (theme) => ({
```
Comment (Rajdip019) [inline ui/components/MeshAdapterConfigComponent.js:18]: Can't we just remove these commented codes?

File: ui/components/MeshAdapterConfigComponent.js [hunk 2]
```diff
@@ -63,110 +115,111 @@ const styles = (theme) => ({
...
-    // if(meshAdapters.sort().join(',') !== state.meshAdapters.sort().join(',')){
-    if (meshAdaptersts > state.ts) {
-      return {
-        meshAdapters,
-        ts : meshAdaptersts
-      };
+const MeshAdapterConfigComponent = (props) => {
+
+  const labelRef = useRef(null)
+  // this.state = {
```
Comment (Rajdip019) [inline ui/components/MeshAdapterConfigComponent.js:129]: Same here we can directly remove these commented codes. Or specify why these are commented.

**Comment by Rajdip019** [comment]
Also @EraKin575 can you give a video of these components as there are a lot of changes.

**Comment by EraKin575** [comment]
@Rajdip019 

https://github.com/meshery/meshery/assets/100225222/10ac36a3-867a-417b-8767-05f8f4f00be5

**Comment by leecalcote** [comment]
Thank you, @EraKin575 👏

**Review by Rajdip019 — APPROVED** [review]

**Review by aabidsofi19 — APPROVED** [review]

**Pull request merged**